docs/runbook/formulare-support.md

```diff
@@ -136,7 +136,37 @@ Treba kontaktovať vlastníkov formuláru na strane magistrátu. Ak ide o závä
 
 ### Co robit ked formular skonci v SHAREPOINT_ERROR?
 
-Najlepsie je pingnut Řehulka Erik a on ti povie ako dalej. Ked sa to od neho dozvies, napis to sem aby sme vedeli.
+Vo veľkej väčšine prípadov je to z toho dôvodu, že používateľ zadal do niektorého poľa dátum s rokom pod 1900. Sharepoint z nejakého dôvodu toto neakceptuje ako valídny rok, preto toto treba upraviť a zopakovať odoslanie. V prvom rade je teda fajn si pozrieť log toho erroru, ak je tam v dátach naozaj dátum s rokom pod 1900, tak je to jasné, v opačnom prípade je treba zreprodukovať odosielanie, čo je popísané nižšie.
```

Comment:
I would replace "pod 1900" with "menej ako 1900" in all the text from a grammar perspective.